code/game/gamemodes/clock_cult/clock_helpers/resource_helpers.dm

```diff
@@ -0,0 +1,19 @@
+//Helpers for potential and wisdom.
+
+//Returns true if the potential is greater than or equal to a set amount, and false otherwise.
+/proc/has_clockwork_potential(amt)
```

Comment:
```#define```s, this allows them to be named (```has_clockwork_potential()```) but no proc call overhead.
Procs that are simple (such as these) or hot (called a lot) should always be ```#define```s

Defines are also always(mostly) capitalised, ```HAS_CLOCKWORK_POTENTIAL(amt)```